### ISSUE TITLE
performance of SemanticKitti pre-trained model

### ISSUE THREAD

**Issue by lqzhao** [xinge008/Cylinder3D#30]
![image](https://user-images.githubusercontent.com/43566465/110816151-0ab98d00-82c6-11eb-80b9-ca98f0c268e5.png)
Hi, I just ran an evaluation using the pre-trained model of SemanticKitti, the results seem to be very low, compared with the number in the paper. That's so strange. I'm using torch1.3, is it a bug about the version of Pytorch? Did anyone meet this before? @xinge008

**Comment by xinge008** [comment]
It is wired. I run the evaluation on Validation set again, and the results are normal (about mIoU=65). I have no idea about your issue. Perhaps the version of PyTorch or Spconv makes difference. BTW, do not forget to set the evaluation mode and run the evaluation at the beginning if the pretrained model is loaded.

**Comment by lqzhao** [comment]
![image](https://user-images.githubusercontent.com/43566465/110901880-f3b78100-833f-11eb-9e07-6334bf543f1a.png)
After training for almost one epoch, the performance goes better and better, it has reached 65 mIoU. That's interesting.

**Comment by abhigoku10** [comment]
@lqzhao  i think its due to the torch version we get like this i also had faced the same issue but now it got resolved

**Comment by lqzhao** [comment]
> ut now it got resolved

Thanks,  what version should I use to avoid this? @abhigoku10

**Comment by lixiang-ucas** [comment]
have you figured out the problem?@lqzhao

**Comment by vaydingul** [comment]
> have you figured out the problem?@lqzhao

As @xinge008 mentioned,

Inferencing in evaluation mode directly at the beginning of the training yields the desired results. (In my case) @lixiang-ucas @lqzhao

**Comment by owoshch** [comment]
Hi @lqzhao! I'm trying the reproduce the results on the validation set. Can you please guide me on the scripts you used to get the metrics you reported?

**Comment by lqzhao** [comment]
> Hi @lqzhao! I'm trying the reproduce the results on the validation set. Can you please guide me on the scripts you used to get the metrics you reported?

Actually, I just followed the instructions the author provided and reproduced the results successfully. What's the specific issue in your case?

**Comment by GaloisWang** [comment]
excuse me, when I test the pretrained model on KITTI 08 sequence, the results are all 0. Have you encountered this problem before？
